src/main/java/view/DashboardView.java

```diff
@@ -1,18 +1,16 @@
 package view;
 
 // Imports for dashboard
+
+import interface_adapters.Buy.BuyViewModel;
 import interface_adapters.Dashboard.DashboardController;
 import interface_adapters.Dashboard.DashboardState;
 import interface_adapters.Dashboard.DashboardViewModel;
-import interface_adapters.ResetBalance.ResetBalanceController;
-import interface_adapters.ViewManagerModel;
-
-// Imports to allow dashboard to switch to other views
-import interface_adapters.Buy.BuyViewModel;
-import interface_adapters.Sell.SellViewModel;
 import interface_adapters.GetNews.GetNewsViewModel;
-import interface_adapters.ResetBalance.ResetBalanceViewModel;
```

Comment:
can the ResetBalanceViewModel.java file be deleted as well, since ResetBalance has no view?

Reply:
ok, will make an issue for it